spec/system/unaccompanied_minor_spec.rb

```diff
@@ -951,4 +920,27 @@
       expect(page).to have_field("Refugee travel document number", with: "XYZ123987456")
     end
   end
+
+  def navigate_to_nationality
+    new_application = UnaccompaniedMinor.new
+    new_application.save!
+
+    page.set_rack_session(app_reference: new_application.reference)
+
+    visit "/sponsor-a-child/task-list"
+    expect(page).to have_content(task_list_content)
+
+    click_link("Additional details")
+    expect(page).to have_content("Do you have any of these identity documents?")
+
+    choose("Passport")
+    fill_in("Passport number", with: "123456789")
+    click_button("Continue")
+
+    fill_in("Day", with: "6")
```

Comment:
Nice - I like the refactor.
Is this a date of birth entry? If so, there is a new helper :wink:
`uam_fill_in_date_of_birth(date ....)`